convert-stage-4.sh

```diff
@@ -25,24 +27,27 @@ Usage: $0 [options]
 
     Examples:
 
-        ./mender-convert install-mender-to-mender-disk-image
+        ./mender-convert from_raw_disk_image
```

Comment:
`install-mender-to-mender-disk-image` is one of the so called expert options in `mender-convert` tool. These expert options recently have been hidden by @mirzak. The code hasn't been removed, but they are not listed in `--help` message any more. Besides that `from-raw-disk-image` option requires `--raw-disk-image` CLI argument.
I would leave `install-mender-to-mender-disk-image`  in this place.

Reply:
I don't understand, `install-mender-to-mender-disk-image` option seems to have been completely removed. Looking at the `mender-convert` main script, I can see that passing such option will just show help and exit. Then I think we should just delete this example from the documentation

@mirzak can you please clarify?

Reply:
The user facing options have been removed, https://github.com/mendersoftware/mender-convert/commit/024ee0efd91431249e0eab21e6b7217f6f0a4f1d

The underlying code is still  there because it is used by the `fram_raw_disk_image` execution path, but there is no way to call that method invoking the `mender-convert` tool.

Reply:
Correct. Then I will remove the example from the help text